src/hooks/useWindowDimensions.ts

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from "react";
+
+export function useWindowDimensions() {
```

Comment:
Can we use https://v5.mantine.dev/hooks/use-viewport-size/ for that instead?

Reply:
yea, totally forgot about this one. will do.